survey.html

```diff
@@ -30,35 +30,66 @@
 
                 <h1>Survey</h1>
 
-                <form action="survey.cgi" method="POST">
-                    What is your name?

```

Comment:
Missing labels
Control/label pairs not wrapped in `<p>` as per lab instructions